package.json

```diff
@@ -5,16 +5,23 @@
   "author": "",
   "license": "",
   "scripts": {
+    "build": "webpack",
```

Comment:
"webpack -d --watch" , should add a watch statement so that it continually updates

Reply:
Only change recommended

Reply:
Added watch script.